### PR TITLE
Converted to paver from fab

### PR DESCRIPTION
**Pull request by Muddasser**

### CODE REVIEW

**Comment by Muddasser** [comment]
@raeeschachar @benpatterson Please review.

File: pavelib/paver_consts.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+# Paths for different directories
+LOG_DIR = path('./logs').abspath()
+TEST_DIR = path('./stage/tests/').abspath()
+REPORT_DIR = path('./reports').abspath()
+SCREENSHOT_DIR = path('./screenshots').abspath()
+BASELINE_DIR = path('./certs/screenshots/baseline').abspath()
+
+# XML reports names
+SMOKE_TEST_REPORT = '/smoke_tests_results.xml'
```
Comment (benpatterson) [inline pavelib/paver_consts.py]: nit: don't use the directory indicator with this one (the forward slash).

**Comment by benpatterson** [comment]
This looks like it has some of Raees's changes as well. Could you split it out? One way to do that is to make sure you have Raees's commit, and then make the PR against Raees's branch instead of master.

I gave it a quick look but would need to give it more in-depth look. So far it looks straightforward.

File: pavelib/paver_consts.py (hunk 1 above)
Comment (Muddasser) [inline pavelib/paver_consts.py]: Done.

**Comment by Muddasser** [comment]
@benpatterson

**Pull request closed without merging**